gspread/worksheet.py

```diff
@@ -485,13 +485,23 @@ def get_values(self, range_name=None, combine_merged_cells=False, **kwargs):
         """
         try:
             vals = fill_gaps(self.get(range_name, **kwargs))
+
             if combine_merged_cells is True:
                 spreadsheet_meta = self.spreadsheet.fetch_sheet_metadata()
                 worksheet_meta = finditem(
                     lambda x: x["properties"]["title"] == self.title,
                     spreadsheet_meta["sheets"],
                 )
-                return combined_merge_values(worksheet_meta, vals)
+                grid_range = a1_range_to_grid_range(
+                    get_a1_from_absolute_range(range_name),
+                )
```

Comment:
regarding above comment I would suggest we check if range_name is `None` or not. then call the absolute or not.

I would suggest something like a filter pattern:

```python
if range_name is no None:
  range_name = get_a1_from_absolute_range(range_name)

grid_range = a1_range_to_grid_range(range_name)
```